File: CPP/Program/Product.cpp

#ifndef PRODUCT_H
#define PRODUCT_H
#include <string>

class Product {
private:
    std::string idProduct;
    std::string name;
    std::string brand;
    int price;

public:
    Product(std::string idProduct = "", std::string name = "", std::string brand = "", int price = 0) {
        this->idProduct = idProduct;
        this->name = name;
        this->brand = brand;
        this->price = price;
    }

    std::string get_IdProduct() {
        return idProduct;
    }

    void set_IdProduct(std::string idProduct) {
        this->idProduct = idProduct;
    }

    std::string get_Name() {
        return name;
    }

    void set_Name(std::string name) {
        this->name = name;
    }

    std::string get_Brand() {
        return brand;
    }

    void set_Brand(std::string brand) {
        this->brand = brand;
    }

    int get_Price() {
        return price;
    }

    void set_Price(int price) {
        this->price = price;
    }
};
#endif

File: PYTHON/Program/Product.py
class Product:
    def __init__(self, idProduct="", name="", brand="", price=0):
        self.idProduct = idProduct
        self.name = name
        self.brand = brand
        self.price = price

    def get_IdProduct(self):
        return self.idProduct

    def set_IdProduct(self, idProduct):
        self.idProduct = idProduct
        
    def get_Name(self):
        return self.name

    def set_Name(self, name):
        self.name = name

    def get_Brand(self):
        return self.brand

    def set_Brand(self, brand):
        self.brand = brand

    def get_Price(self):
        return self.price

    def set_Price(self, price):
        self.price = price